### DISCUSSION TITLE
KDV Oran Değişikleri

### DISCUSSION THREAD

**Discussion by perakasma** [mlevent/fatura#52] [General]
Merhaba Mert Bey'e bu kapsamlı proje için tekrardan teşekkür ederim.
Bilindiği gibi KDV oranlarında bazı değişikler oldu: %8 olan oran %10; %18 olan oran ise %20 olarak yükseltildi. Acaba bu değişikliğin ilgili projede bir yanlışlığa neden olması söz konusu olur mu ? Dolayısıyla proje şu an güncelliğini hâla koruyor mu ? 
Teşekkür ederim.

**Comment by mlevent** [comment]
Merhabalar @perakasma,
vergi oranlarında yapılan değişikliklerle ilgili geçen ay bir güncelleme(#48) ile yeni versiyon yayınlamıştım; https://github.com/mlevent/fatura/releases/tag/v0.2.2 
Paket şu an güncel.
Kolaylıklar dilerim, iyi çalışmalar.

**Comment by perakasma** [comment]
Hızlı yanıtınız için teşekkür ederim. Son olarak bir şey sormak istiyorum.
composer ile kurulum yapıyorum. Ancak dosya boyutu olağanüstü şekilde artmış görünüyor. vendor klasörü 2 kusür MB'yken şu an 24 MB olmuş durumda. En büyük artış"guzzlehttp" klasöründen kaynaklanıyor. Önceki sürümlerdeki guzzlehttp klasörünü kopyalamak ileride hataya neden olur mu sizce ? Hatta dahası sadece "mlevent" klasörünü güncellemek yeterli olur mu ? 
İlginiz için tekrardan teşekkür ederim. Saygılarımla

**Comment by mlevent** [comment]
> Hızlı yanıtınız için teşekkür ederim. Son olarak bir şey sormak istiyorum.
> 
> composer ile kurulum yapıyorum. Ancak dosya boyutu olağanüstü şekilde artmış görünüyor. vendor klasörü 2 kusür MB'yken şu an 24 MB olmuş durumda. En büyük artış"guzzlehttp" klasöründen kaynaklanıyor. Önceki sürümlerdeki guzzlehttp klasörünü kopyalamak ileride hataya neden olur mu sizce ? Hatta dahası sadece "mlevent" klasörünü güncellemek yeterli olur mu ? 
> 
> İlginiz için tekrardan teşekkür ederim. Saygılarımla

Aslında bahsettiğiniz gibi bir artış görünmüyor dosya boyutlarında. Paketi kaldırıp tekrar temiz kurulum yaparsanız normale dönebilir.

**Comment by perakasma** [comment]
Evet, haklısınız. Sorun bennden kaynaklıymış: php.ini dosyasında extension=zip uzantısının devre dışı olmasından kaynaklanıyormuş. Sorunu çözdüm. Teşekkür ederim.
